Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,432 @@
+                                              DeviceEventCategory == 4, "Blocked Web Sites (4)",
+                                              DeviceEventCategory == 8, "Blocked Java Etc. (8)",
+                                              DeviceEventCategory == 16, "User Activity (16)",
+                                              DeviceEventCategory == 32, "Attacks (32)",
+                                              DeviceEventCategory == 64, "Dropped TCP (64)",
+                                              DeviceEventCategory == 128, "Dropped UDP (128)",
+                                              DeviceEventCategory == 256, "Dropped ICMP (256)",
+                                              DeviceEventCategory == 512, "Network Debug (512)",
+                                              DeviceEventCategory == 1024, "Connection Closed (1024)",
+                                              DeviceEventCategory == 2048, "Dropped LAN TCP (2048)",
+                                              DeviceEventCategory == 4096, "Dropped LAN UDP (4096)",
+                                              DeviceEventCategory == 8192, "Dropped LAN ICMP (8192)",
+                                              DeviceEventCategory == 32768, "Modem Debug (32768)",
+                                              DeviceEventCategory == 65536, "VPN Tunnel Status (65536)",
+                                              DeviceEventCategory == 131072, "IEEE 802.11 Management (131072)",
+                                              DeviceEventCategory == 262144, "Connection Opened (262144)",
+                                              DeviceEventCategory == 524288, "System Environment (524288)",
+                                              DeviceEventCategory == 1048576, "Expanded - VoIP Activity (1048576)",
+                                              DeviceEventCategory == 2097152, "Expanded - WLAN IDS Activity (2097152)",
+                                              DeviceEventCategory == 4194304, "Expanded - SonicPoint Activity (4194304)",
+                                              DeviceEventCategory == 8388608, "Expanded - Unified Policy Engine (8388608)",
+                                              "Legacy Category Not Mapped"
+                                              )
+  | extend IPSPriority = case(ipspri == 1, "High (1)",
+                                              ipspri == 2, "Medium (2)",
+                                              ipspri == 3, "Low (3)",
+                                              ""
+                                              )
+  | extend AntiSpywarePriority = case(spypri == 1, "High (1)",
+                                              spypri == 2, "Medium (2)",
+                                              spypri == 3, "Low (3)",
+                                              ""
+                                              )
+  | extend
+      EventProduct = "SonicOS"

```

Comment:
is this name as per vendor documentation?

Reply:
Yes, that's correct. Our firewall's operating system is SonicOS.